androidApp/src/main/java/press/home/HomeView.kt

```diff
@@ -113,10 +124,58 @@ class HomeView @AssistedInject constructor(
     )
   }
 
+  private val createEditorView = { noteUuid: Uuid ->
+    editorViewFactory.create(
+        context = context,
+        openMode = ExistingNote(noteUuid),
+        onDismiss = { notesList.collapse() }
+    )
+  }
+
+  private var noteForEditor by observable<NoteParcelable?>(null) { _, oldNote, newNote ->
```

Comment:
I like this idea of using an `observable` delegate to show `EditorView`. I do think that here `NoteParcelable?` can be replaced with just `Note?` `NoteParcelable` should be kept only for saving and restoring of state. Other parts of HomeView should remain oblivious to it.